online-workflow/src/main/java/eu/itesla_project/online/StateAnalyzer.java

```diff
@@ -442,7 +423,13 @@ private boolean computePostContingencyState(Network network, String stateId, Con
                 // run load flow on post contingency state
                 logger.info("{}: running load flow on post contingency state {}", stateId, postContingencyStateId);
                 LoadFlowResult result = loadFlow.run();
-                if ( result.isOk() ) {
+                // store post contingency state
+                Integer stateIdInt = Integer.parseInt(stateId);
+                if (parameters.storeStates() || ((parameters.analyseBasecase()) && (stateIdInt == 0))) {
```

Comment:
Why the type of stateIdInt is Integer instead of int. Is stateId always a string representing a number:
- If not it will throw a NumberFormatException.
- Otherwise, stateId could already be an integer.